src/components/form/GitNewBranchForm.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <q-form
+    ref="form"
+    @submit="onSubmit"
+    data-cy="git-newBranch-form"
+    class="q-gutter-md git-newBranch-form">
+    <q-input
+      filled
+      v-model="newBranch"
+      :label="$t('page.modelizer.git.newBranch.branch')"
+      lazy-rules
+      data-cy="git-branch-input"
+      :rules="[(v) => notEmpty(t, v), (v) => isDuplicateBranchName(t, branches, v)]"
+    />
+    <q-checkbox
+      v-model="checkout"
+      data-cy="git-checkout-checkbox"
+      :label="$t('page.modelizer.git.newBranch.checkout')"
+    />
+
```

Comment:
why empty line ?

Reply:
add empty to line 14 too, to be more readable :)